contrib/cirrus/runner.sh

```diff
@@ -134,28 +229,19 @@ dotest() {
         # does not return
     fi
 
-    # 'logformatter' script makes test logs readable; only works for some tests
-    case "$testsuite" in
-        integration|system)  output_filter=logformatter ;;
-        *)                   output_filter="cat"        ;;
-    esac
-
-    # containers/automation sets this to 0 for it's dbg() function
+    # containers/automation sets this to 0 for its dbg() function
     # but the e2e integration tests are also sensitive to it.
     unset DEBUG
 
     # shellcheck disable=SC2154
+    local localremote="$PODBIN_NAME"
     case "$PODBIN_NAME" in
-        podman)
-            # ginkgo doesn't play nicely with C Go
-            make local${testsuite} \
-                |& "$output_filter"
-            ;;
-        remote)
-            make remote${testsuite} PODMAN_SERVER_LOG=$PODMAN_SERVER_LOG \
-                |& "$output_filter"
-            ;;
+        podman)  localremote="local" ;;
     esac
+
+    # ginkgo doesn't play nicely with C Go (FIXME: what does this comment mean?)
```

Comment:
@cevich can you illuminate this comment? If it's important, I'd like to reword it in such a way that it will help someone like me.

Reply:
I think this comment can be deleted, it was documenting something that IIRC was fixed in the Makefile instead.

Reply:
Thank you! Removed.